File: app/src/main/java/io/euphoria/xkcd/app/connection/event/MessageEvent.java
package io.euphoria.xkcd.app.connection.event;

import io.euphoria.xkcd.app.data.Message;

/* Created by Xyzzy on 2017-02-24. */

/* Event encapsulating a single (potentially fresh) message */
public interface MessageEvent extends ConnectionEvent {

    /* The message posted */
    Message getMessage();

}

File: app/src/main/java/io/euphoria/xkcd/app/connection/event/MessageEvent.kt
package io.euphoria.xkcd.app.connection.event

import io.euphoria.xkcd.app.data.Message

/** Created by Xyzzy on 2017-02-24.  */ /* Event encapsulating a single (potentially fresh) message */
open interface MessageEvent : ConnectionEvent {
    /* The message posted */
    val message: Message
}
